apps/messagelist/app.js

```diff
@@ -24,6 +24,9 @@
     RIGHT = 1, LEFT = -1, // swipe directions
     UP = -1, DOWN = 1;    // updown directions
   const Layout = require("Layout");
+  const debug = function() {
+    if (global.DEBUG_MESSAGELIST) console.log.apply(console.log, ['messagelist:'].concat(arguments));
```

Comment:
`console.log` doesn't use its `this` parameter, so while this doesn't have an effect here, we want to apply the function to the object it lives on:
```suggestion
    if (global.DEBUG_MESSAGELIST) console.log.apply(console, ['messagelist:'].concat(arguments));
```

Reply:
Thanks!